.github/workflows/uberjar.yml

```diff
@@ -23,16 +25,63 @@ on:
         description: "Optional full-length commit SHA-1 hash"
 
 jobs:
+  get-container-info:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    name: Get container info
+    outputs:
+      editions: ${{ toJson(fromJson(steps.info.outputs.result).editions) }}
+      ee: ${{ toJson(fromJson(steps.info.outputs.result).ee) }}
+      oss: ${{ toJson(fromJson(steps.info.outputs.result).oss) }}
+    steps:
+      - name: Get docker repo and tag
+        id: info
+        uses: actions/script@v7
```

Comment:
```suggestion
        uses: actions/github-script@v7
```

Reply:
wow. super weird [this actually worked](https://github.com/metabase/metabase/actions/runs/14673089003/job/41183912180?pr=57298). I guess I found a hidden alias? In any event, will use the right one.